Week-8/Graph.js

```diff
@@ -0,0 +1,184 @@
+class MyGraph {
+    constructor() {
+        this.adjacencyList = {};
+    }
+
+    addVertex(vName) {
+        if (this.adjacencyList[vName]) {
+            throw 'Vertex already exists';
+        }
+        this.adjacencyList[vName] = new Set();
+    }
+
+    addEdge(v1, v2) {
+        if (!this.adjacencyList[v1]) {
+            throw `${v1} doesn't exist`;
+        }
+        if (!this.adjacencyList[v2]) {
+            throw `${v2} doesn't exist`;
+        }
+        this.adjacencyList[v1].add(v2);
+        this.adjacencyList[v2].add(v1);
+    }
+
+    removeEdge(v1, v2) {
+        if (!this.adjacencyList[v1]) {
+            throw `${v1} doesn't exist`;
+        }
+        if (!this.adjacencyList[v2]) {
+            throw `${v2} doesn't exist`;
+        }
+        this.adjacencyList[v1].delete(v2);
+        this.adjacencyList[v2].delete(v1);
+    }
+
```

Comment:
Give one line between a chunk of code ie if else loop functions variable declaration etc,